recipes/flytekit/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "flytekit" %}
+{% set version = "0.26.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flytekit-{{ version }}.tar.gz
+  sha256: 83bf9ed9d014b0683f49a668113266f4a2f394d4870cd733a9b73b438fe8ef23
+
+build:
+  # Skipping Windows for now
+  # xref: conda-forge/protobuf-feedstock#133
+  skip: true  # [py2k or win]
```

Comment:
Is the only reason for this not being `noarch` the skip? In that case you can leave it `noarch` and restrict it by using the `__unix` virtual package.

Reply:
Edit: Scratch that, just saw that you have `singledispatchmethod  # [py<38]` so it cannot be `noarch`.